assembly/assembly-machine-war/src/main/java/org/eclipse/che/ide/ext/java/server/ApiEndpointAccessibilityChecker.java

```diff
@@ -46,18 +48,36 @@ public ApiEndpointAccessibilityChecker(@Named("api.endpoint") String apiEndpoint
     }
 
     @PostConstruct
-    public void start()
-            throws ForbiddenException, BadRequestException, ConflictException, NotFoundException,
-                   ServerException, UnauthorizedException, IOException {
-        final HttpJsonResponse pingResponse = httpJsonRequestFactory.fromUrl(apiEndpoint)
-                                                                    .setMethod(HttpMethod.GET)
-                                                                    .setTimeout(2000)
-                                                                    .request();
-        if (pingResponse.getResponseCode() != HttpURLConnection.HTTP_OK) {
-            throw new RuntimeException(
-                    "The workspace agent that we inject into your workspace machine has failed to start due to a DNS resolution error. " +
-                    "This error happens when the agent cannot resolve the location of Che due to unusual configuration. " +
-                    "Please post details of your configuration in an issue on Che's GitHub page.");
+    public void start() {
+        try {
+            final HttpJsonResponse pingResponse = httpJsonRequestFactory.fromUrl(apiEndpoint)
+                    .setMethod(HttpMethod.GET)
+                    .setTimeout(2000)
+                    .request();
+            if (pingResponse.getResponseCode() == HttpURLConnection.HTTP_OK) {
+                return;
+            }
+        }
+
+        catch (ApiException | IOException e) {
+            LOG.error( e.getLocalizedMessage(), e);
         }
+
+        LOG.error("The workspace agent has attempted to start, but it is unable to ping the Che server at " + apiEndpoint + ". ");
```

Comment:
Use special API of logger LOG.error("some string {} with {} parameters", firstParam, secondParam);